src/scss/components/carousel.scss

```diff
@@ -9,9 +9,9 @@
   }
 
   // Slides
-  .item {
```

Comment:
For the carousel work could you take a look at the margin at the top of the carousel (See .carousel class).
You could probably set this this to just '-$line-height-computed' which comes out as around 2.4rem (24px).
It actually is needed to compensate for the nav margin which is currently set to 20px in starter.

Reply:
I think we also need to set z-index on the carousel as it currently sits on top of the lower edge of the nav menu. I've tried z-index: -2 which appears to fix it but breaks the hover.

Reply:
Done! Also, I was wondering about the breaking hover effect, is that the navbar hover? It seems to be working fine for me

Reply:
This may have something to do with the z-index not being set on the navbar so I can take a look as part of the PR I'm working on.

Reply:
Seems like this has affected the carousel, now we have no access to the controls, so have reverted this change and will see if the changes from your base branch could help remedy this